README.md

```diff
@@ -43,7 +43,25 @@ For example, it could be `.github/workflows/action.yml`
 
 ## Spellcheck Configuration File
 
-The file can be named:
+You can either provide a path to the configuration file or save a file in the root of your repository with a predefined name (list below). If `configPath` is provided then it will be used and the other configuration options will be ignored. If `configPath` is not provided then the repository is searched after a first match
+
+Example:
+```yaml
+name: Spellcheck Action
+on: push
+jobs:
+  build:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: rojopolis/spellcheck-github-actions@master
+      name: Spellcheck
+      with:
+        configPath: config/.spellcheck.yml # put path to configuration file here
```

Comment:
We need to adjust the example in the documentation to adhere with the recent name changes